jdaviz/configs/default/tests/test_aida.py

```diff
@@ -41,6 +41,9 @@ def test_get_viewport_sky(imviz_helper, image_hdu_wcs):
     assert_coordinate_close(viewport['center'], expected_center)
     assert_angle_close(viewport['fov'], expected_fov)
 
+    expected_rotation = -0.00052851 * u.deg
```

Comment:
My personal apologies for the jdaviz rotation framework that return torturously close-to-but-still-non-zero numbers here.